src/java.base/share/classes/java/io/Reader.java

```diff
@@ -140,6 +140,125 @@ public void close() {
         };
     }
 
+    /**
+     * Returns a {@code Reader} that reads characters from a
+     * {@code CharSequence}, starting at the first character in the sequence.
```

Comment:
I see Alan and I have different views on the inclusion of the details.
With only a single CharSequence argument, I can't see any ambiguity about the starting position.

Reply:
> I see Alan and I have different views on the inclusion of the details. With only a single CharSequence argument, I can't see any ambiguity about the starting position.

It could be a second sentence, "Reading starts at the first character in the sequence".

Reply:
Done. Adopted Alan's wording.